### PR TITLE
Re-re-open PR

### PR DESCRIPTION
**Pull request by william-vw**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/N3/pull/191.html" title="Last updated on Jun 30, 2023, 8:02 PM UTC (56c34ae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/N3/191/770ce83...56c34ae.html" title="Last updated on Jun 30, 2023, 8:02 PM UTC (56c34ae)">Diff</a>

### CODE REVIEW

**Review by domel — APPROVED** [review]

File: spec/semantics.html [hunk 1]
```diff
@@ -670,10 +673,10 @@ <h2>Graphs with variables</h2>
...
   <!-- unless those domain elements are also language terms ... -->
-  If <i>not</i> applying the valuation function to the graph term, $Q_I$ will have to map the <a>graph term</a> <code>&lt;{}, {},  {(x, :is, :wise)&gt;}</code>,
+  If <i>not</i> applying the valuation function to the graph term, $Q_I$ will have to map the original <a>graph term</a> <code>&lt;{}, {},  {(x, :is, :wise)&gt;}</code>,
   which is however no longer <a>ground</a> as its graph is no longer <a>closed</a>
-  (`x` is quantified outside of the graph). In our <a>basic interpretation</a>, the mapping $Q_I$ is only defined for <a>ground</a> <a>graph terms</a>.
-  We thus need to first ground these kinds of <i>leaked variables</i>, 
+  (`x` is quantified <i>outside of the graph</i>). In our <a>basic interpretation</a>, the mapping $Q_I$ is only defined for <a>ground</a> <a>graph terms</a>.
+  Hence, in the latter case, we first need to ground these kinds of <i>leaked variables</i>, 
   i.e., variables that become free when considering the graph term in isolation (as is required when determining isomorphisms).
```
Comment (TallTed) [inline spec/semantics.html:680]:
```suggestion
  i.e., variables that become free when considering the graph term in isolation (as is required when determining isomorphism).
```

File: spec/semantics.html [hunk 2]
```diff
@@ -763,7 +772,7 @@ <h2></h2>
       </section>
       <section>
         <h2>Basic entailment</h2>
-The previous definition allows us to introduce the notion of basic entailment:
+The previous definition allows us to introduce the notion of basic entailment:~
```
Comment (TallTed) [inline spec/semantics.html:775]: If this tilde was intentional, please explain it, at least in comment.
```suggestion
The previous definition allows us to introduce the notion of basic entailment:
```

File: spec/semantics.html [hunk 3]
```diff
@@ -965,12 +974,7 @@ <h2>Constraints for `log:implies`</h2>
         <p>Let $G$ and $H$ be two graphs, we say that $G$ <dfn data-lt="log-entailment|log entailment">log-entails</dfn> $H$, written as $G ⊨_{log} H$,
         if and only if  every interpretation $I$ that <a>log satisfies</a> $G$ ($I ⊨_{log} G$) also <a>log satisfies</a> $H$ ($I ⊨_{log} H$).</p> 
 
-        To illustrate the definition we discuss some examples of entaiment and non-entialment in the sections below.
-
-     
-
-        
-
+        To illustrate the definition we discuss some examples of entaiment and non-entailment in the sections below.
```
Comment (TallTed) [inline spec/semantics.html:977]:
```suggestion
        To illustrate the definition, we discuss some examples of entailment and non-entailment in the sections below.
```

File: spec/semantics.html [hunk 4]
```diff
@@ -1054,12 +1058,12 @@ <h2>Constraints for `log:implies`</h2>
 </section>
         <section> <h3>Unquoting</h3>
          <p> A special form of entailment enabled by our semantics is the so-called <dfn data-lt="unquote">unquoting</dfn>. By that, we mean the 
-          entailment which enables us to make the graph conatined in an graph term true. In order to illustrate the idea, we give an example. </p>
+          entailment which enables us to make the graph contained in a graph term true. In order to illustrate the idea, we give an example. </p>
```
Comment (TallTed) [inline spec/semantics.html:1061]:
```suggestion
          entailment which enables us to make the graph contained in a graph term true. To illustrate the idea, we give an example. </p>
```

**Comment by TallTed** [comment]
Please give this PR a better title, that actually describes what it's trying to do.

**Comment by TallTed** [comment]
@william-vw -- Please give this PR a better title, that actually describes what it's trying to do, which is not "re-re-open itself".

**Pull request merged**